src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanBuildStep.java

```diff
@@ -264,6 +264,13 @@ private Map<String, String> getScanProperties(Run<?,?> build, TaskListener liste
 			properties.put("APPSCAN_CLIENT_VERSION", Jenkins.VERSION);
 			properties.put("IRGEN_CLIENT_PLUGIN_VERSION", JenkinsUtil.getPluginVersion());
 			properties.put("ClientType", JenkinsUtil.getClientType());
+            String fetchServer = m_authProvider.getServer();
+            properties.put("serverURL",fetchServer);
+            if(!(fetchServer == null || fetchServer.equals(""))){
+                if(!fetchServer.contains("appscan.com")){
+                    properties.put("certificates",Boolean.toString(m_authProvider.getCertificates()));
+                }
```

Comment:
I suggest using a property name other than "certificates", since it's not really clear what that means. A property name like "acceptssl" would be more clear as to what the purpose is.
Also, why do we need to check the server value here? Couldn't we just check m_authProvider.getCertificates() to see if this property should be set?

Reply:
Okay, Now, i am using "acceptInvalidCerts" string in place of the earlier one.

Matt, checking for the server value because I am putting the value of the " acceptInvalidCerts" only when there is AppScan 360 server. 
If an ASoC user with checked "Allow Untrusted Connection" tends to build the scan, it also get appended in the property and In the SA Client class, it can append the "-Dacceptssl" in the APPSCAN_OPTS.